modules/bvl_feedback/README.md

```diff
@@ -0,0 +1,15 @@
+# Behavioural Feedback
+
+
+## Purpose
+
+The Feedback module allows users to flag and comment on any database field particularly for behavioural data entry. 
+Two default feedback types exist: input and scoring errors. Additional feedback types can be added to feedback_bvl_type table.
+
+## Behavioral Feedback Notifications
+
+The Dashboard Widget tracks issues and corrections to behavioural/clinical data points. Notification are clickable and redirect to the instrument with new changes or feedback.
```

Comment:
This should be in an interactions section describing how it registers widgets on the dashboard, the description of being clickable is unnecessary for the spec. It's an incidental part of the implementation, not a core part of the design.